src/server/pingserver-rs/benches/benchmark.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use criterion::{criterion_group, criterion_main, Criterion, Throughput};
+use pelikan_pingserver_rs::PingserverBuilder;
+
+use std::io::{Read, Write};
+use std::net::TcpStream;
+use std::time::Duration;
+
+fn pingserver_benchmark(c: &mut Criterion) {
+    // launch the pingserver and wait until its ready
+    let pingserver = PingserverBuilder::new(None).spawn();
+    std::thread::sleep(Duration::from_millis(100));
```

Comment:
Is there a chance that this step is flaky on a very slow, shared testing platform?

Reply:
Yeah - I'll increase that - a few seconds should be plenty for the server to startup, 100ms could wind up as flakey in the future.